redisinsight/api/src/modules/workbench/providers/workbench-commands.executor.ts

```diff
@@ -69,10 +69,12 @@ export class WorkbenchCommandsExecutor {
       nodeOptions,
       mode,
     } = dto;
-
-    const [command, ...commandArgs] = splitCliCommandLine(commandLine);
+    let command = commandLine;
```

Comment:
Why you assigned entire command line to command?

Reply:
Because if we can't parse command in my opinion we still have to send something in analytics.

Reply:
We should not send everything. Never. There might be sensitive information.

Reply:
That mean we have to create another try catch block without analytics where we will return only errorResults?